taskcluster/requirements.in

```diff
@@ -1 +1 @@
-taskcluster-taskgraph>=11.1.0
+taskcluster-taskgraph>=13.0.0
```

Comment:
I think these requirements files (and corresponding env in .taskcluster.yml) should get deleted since we're using the decision image that has Taskgraph already baked in.

These files are only needed if the Decision task has additional dependencies on top of Taskgraph. As it stands, whatever version is defined here, will overwrite the version defined in the image tag (which doesn't cause problems, but could be confusing).

Reply:
We do end up using this actually, over in https://github.com/mozilla/translations/blob/69148ce37c96761f2843e2c72c41b64fe165af51/Taskfile.yml#L242-L246, which powers some tests and local testing tools. Ultimately we do need something that makes sure the taskgraph used for those is the same as we use in the decision task, but perhaps we ought to put that elsewhere (to avoid overwriting the one in the decision task)?

Reply:
Ah, another option could be to use the `run-task` image rather than the `decision` image. But it's not a big deal and certainly not worth blocking this on.